### ISSUE TITLE
Missing sigmoid in structure_loss? structure_loss中是否缺失了一步sigmoid操作？

### ISSUE THREAD

**Issue by Moddel** [weijun88/F3Net#11]
In line 20 of train.py:
`wbce  = F.binary_cross_entropy_with_logits(pred, mask, reduce='none')`

I believe that pred should be in (0,1), so we should add a sigmoid function. Is it missing here?

train.py  的第20行 pred 应该是(0,1)中的值，所以这里是否漏掉了一个sigmoid函数？

**Comment by Moddel** [comment]
这里是我弄错了，F.binary_cross_entropy_with_logits会对pred进行sigmoid操作，无需在外部归一
